.github/workflows/compatibility.yml

```diff
@@ -11,14 +11,14 @@ jobs:
       fail-fast:    false
       max-parallel: 10
       matrix:
-        woocommerce: [ '4.5.0', '4.6.0', '4.7.0', '4.8.0', '4.9.0', '5.0.0', '5.1.0', '5.2.0', '5.3.0', '5.4.0', '5.5.0', '5.6.0', '5.7.0', '5.8.0', '5.9.0', '6.0.0', '6.1.0-rc.1']
+        woocommerce: [ '4.6.0', '4.7.0', '4.8.0', '4.9.0', '5.0.0', '5.1.0', '5.2.0', '5.3.0', '5.4.0', '5.5.0', '5.6.0', '5.7.0', '5.8.0', '5.9.0', '6.0.0', '6.1.0', '6.1.1', '6.2.0-rc.1']
```

Comment:
As mentioned in the team chat, should we speed up raising this to more versions? My gut feeling is 3 versions for each of our release. 

@jrodger ^

Reply:
@deepakpathania - This does not mean a blocker for your PR. But if we delay its merge a bit until tomorrow (Friday) in case James can provide some feedback, that would be great.

Reply:
Yeah, not an issue, I'd like this to be included in the next GS testing cycle next Tuesday so can wait and discuss internally till then if it makes sense to raise more versions.

Reply:
> discuss internally till then if it makes sense to raise more versions.

That makes sense. Sorry for the confusion here. I will raise this for discussion internally, and we can make change in the next cycle 3.8.0.